parser.y

```diff
@@ -55,14 +57,21 @@ entry: main_func {
   }
   ;
 
-/* TODO: parse stmts */
-main_func: INT MAIN '(' ')' '{' exprs return '}' { $$ = $6; }
+main_func: INT MAIN '(' ')' '{' stmts '}' { $$ = $6; }
   ;
 
-/* TODO: move to stmt */
-return: RETURN exprs ';' {}
+stmts: stmts stmt {
+    auto stmts = $1;
+    stmts.push_back($2);
+    $$ = std::move(stmts);
+  }
+  | epsilon { $$ = std::vector<std::unique_ptr<StmtNode>>{}; }
   ;
 
+stmt: ';' { $$ = std::make_unique<NullStmtNode>(); }
+    | RETURN exprs ';' { $$ = std::make_unique<ReturnStmtNode>($2); }
+    ;
```

Comment:
How about adding support for `expr ';'` together?
It's a valid statement, and we can keep most of our `arithmetic.c` by only adding `;`s.
Note: I'm amused by the fact that the present arithmetic.c is unrelated to "arithmetic" 😆